core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -0,0 +1,30 @@
+<script type="text/ng-template" id="components/explorationFooter">
+  <div class="footer navbar-fixed-bottom oppia-exploration-footer">
+    <div class="row">
+        <div class="col-sm-7">
+          <img ng-src="<[getStaticImageUrl('/general/apple.svg')]>" itemprop="image"
```

Comment:
Don't need itemprop here. (I think that's just used for page snippets, but this image doesn't represent the content of the page.)